incubator/hnc/pkg/controllers/object_controller.go

```diff
@@ -218,6 +218,14 @@ func (r *ObjectReconciler) syncPropagated(ctx context.Context, log logr.Logger,
 		return remove, nil
 	}
 
+	// If an object doesn't exist, assume it's been deleted or not yet created. In this case, we will
+	// return write action and the source instance.
+	exist := inst.GetCreationTimestamp() != v1.Time{}
```

Comment:
See if `inst.GetCreationTimestamp().IsZero()` works here. I don't think we ever figured out why it sometimes works but sometimes wouldn't compile.

Reply:
Looks like it still doesn't work. It has compile errors:
pkg/controllers/object_controller.go:224:38: cannot call pointer method on inst.GetCreationTimestamp()
pkg/controllers/object_controller.go:224:38: cannot take the address of inst.GetCreationTimestamp()

Reply:
Ah well, worth a shot.